benchexec/outputhandler.py

```diff
@@ -737,8 +737,7 @@ def add_result(self, run):
         self.score += result.score_for_task(run.identifier, run.properties, run.category, run.status,
                                             run.multiproperty_statuses)
         #if run.properties:
-        self.max_score += result.score_for_task(run.identifier, run.properties, result.CATEGORY_CORRECT, None,
-                                                run.multiproperty_statuses)
+        self.max_score += result.score_for_task(run.identifier, run.properties, result.CATEGORY_CORRECT, None, {None:None})
```

Comment:
Is `{None:None}` really what the method expects? Why is `None` an allowed property?

Wouldn't it make more sense to forbid `None` and use `{}` as empty parameter?